graphql/ui_plugin_build_baron.go

```diff
@@ -140,7 +151,7 @@ func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnIn
 	jql := t.GetJQL(bbProj.TicketSearchProjects)
 	tickets, source, err = multiSource.Suggest(t)
 	if err != nil {
-		return nil, projectNotFoundError, errors.New(fmt.Sprintf("Error searching for tickets: %s", err.Error()))
+		return nil, bbConfig, errors.Errorf("Error searching for tickets: %s", err.Error())
```

Comment:
why doesn't this wrap the error?

Reply:
I assumed this was purposeful because it's a UI error and this is sometimes more readable but that could be wrong

Reply:
wrapped it, thanks!